lib/ReactCrop.js

```diff
@@ -676,6 +676,7 @@ class ReactCrop extends PureComponent {
           style={imageStyle}
           src={src}
           onLoad={e => this.onImageLoad(e.target)}
+          onError={() => this.props.onError()}
```

Comment:
Thanks but to be consistent with `onImageLoaded` please change the prop name to `onImageError` also just do `onError={this.props.onImageError}` so that it receives the args (an error object)